### PR TITLE
Added node ports to core services and reworked application vars a bit…

### PR DESCRIPTION
**Pull request by Vafilor**
… to differentiate between cloud and local

### CODE REVIEW

File: common/onepanel/base/vars.yaml [hunk 1]
```diff
@@ -21,9 +21,26 @@ database:
...
+    apiHTTPPort:
+      required: true
+      default: 8888
+    apiGRPCPort:
+      required: true
+      default: 8887
+  cloud:
+    uiPath:
+      required: true
+      default: web
```
Comment (rushtehrani) [inline common/onepanel/base/vars.yaml]: let's add more descriptive default values here and below.

File: common/onepanel/base/vars.yaml [hunk 2]
```diff
@@ -21,9 +21,26 @@ database:
...
     required: true
     default: localhost
-  webPort:
-    required: true
-    default: 80
\ No newline at end of file
+  local:
+    webHTTPPort:
+      required: true
+      default: 4200
```
Comment (rushtehrani) [inline common/onepanel/base/vars.yaml]: These are all node ports, so they should be in the 30000-32767 range. I suggest:
```
apiHTTPPort: 32000
apiGRPCPort: 32001
webHTTPPort: 32003
```

File: common/onepanel/base/vars.yaml [hunk 3]
```diff
@@ -21,9 +21,26 @@ database:
...
-  webHost:
+  host:
     required: true
     default: localhost
-  webPort:
-    required: true
-    default: 80
\ No newline at end of file
+  local:
+    webHTTPPort:
```
Comment (rushtehrani) [inline common/onepanel/base/vars.yaml]: should we call this `ui` or `web`?

File: common/onepanel/base/vars.yaml (hunk 2 above)
Comment (Vafilor) [reply common/onepanel/base/vars.yaml]: That's a good point, will do.

File: common/onepanel/base/vars.yaml (hunk 3 above)
Comment (Vafilor) [reply common/onepanel/base/vars.yaml]: Now that you mention it, I'm leaning toward ui. Seems more consistent with api.

File: common/onepanel/base/vars.yaml [hunk 4]
```diff
@@ -21,9 +21,26 @@ database:
     required: true
     default: postgres
 application:
-  webHost:
+  host:
     required: true
     default: localhost
```
Comment (rushtehrani) [inline common/onepanel/base/vars.yaml]: let's change this to `<local-provider-ip>`

File: common/onepanel/base/vars.yaml (hunk 1 above)
Comment (rushtehrani) [reply common/onepanel/base/vars.yaml]: Here are the defaults for these:
uiPath: `/`
apiPath: `/api`
apiGRPCPort: `8887`

Comment (rushtehrani) [reply common/onepanel/base/vars.yaml]: I made this change in another branch

File: common/onepanel/base/vars.yaml (hunk 4 above)
Comment (rushtehrani) [reply common/onepanel/base/vars.yaml]: made this change in another branch

File: common/onepanel/base/vars.yaml (hunk 1 above)
Comment (rushtehrani) [reply common/onepanel/base/vars.yaml]: change made in #22

**Pull request merged**